tests/wdio.conf.ts

```diff
@@ -0,0 +1,260 @@
+import AllureReporter from '@wdio/allure-reporter';
+
+import { getLogs, initLogger, logInfo } from './helpers/browserLogger';
+
+const allure = require('allure-commandline');
+
+// This is deprecated without alternative (https://github.com/nodejs/node/issues/32483)
+// we need it to be able to reuse jitsi-meet code in tests
+require.extensions['.web.ts'] = require.extensions['.ts'];
+
+const chromeArgs = [
+    '--allow-insecure-localhost',
+    '--use-fake-ui-for-media-stream',
+    '--use-fake-device-for-media-stream',
+    '--disable-plugins',
+    '--mute-audio',
+    '--disable-infobars',
+    '--autoplay-policy=no-user-gesture-required',
+    '--auto-select-desktop-capture-source=Your Entire screen',
+    '--no-sandbox',
+    '--disable-dev-shm-usage',
+    '--disable-setuid-sandbox',
+    '--use-file-for-fake-audio-capture=tests/resources/fakeAudioStream.wav'
+];
+
+if (process.env.RESOLVER_RULES) {
+    chromeArgs.push(`--host-resolver-rules=${process.env.RESOLVER_RULES}`);
+}
+if (process.env.ALLOW_INSECURE_CERTS === 'true') {
+    chromeArgs.push('--ignore-certificate-errors');
+}
+if (process.env.HEADLESS === 'true') {
+    chromeArgs.push('--headless');
+    chromeArgs.push('--window-size=1400,600');
+}
+if (process.env.VIDEO_CAPTURE_FILE) {
+    chromeArgs.push(`use-file-for-fake-video-capture=${process.env.VIDEO_CAPTURE_FILE}`);
+}
+
+const chromePreferences = {
+    'intl.accept_languages': 'en-US'
+};
+
+const TEST_RESULTS_DIR = 'test-results';
+
+export const config: WebdriverIO.MultiremoteConfig = {
+
+    runner: 'local',
+
+    specs: [
+        'specs/**'
+    ],
+    maxInstances: 1,
+
+    baseUrl: 'https://alpha.jitsi.net/torture',
```

Comment:
Can you make this overridable via env var pl?

Reply:
There is a very simple way to override stuff from configs. I wanted to leave it so it is a working one without any modifications, the way we have it for make dev.

Reply:
https://webdriver.io/docs/organizingsuites#inherit-from-main-config-file
I will use the same to introduce the Firefox tests. And different environments will be overriding stuff and excluding tests if they need to ... and change the deployment URL.

Reply:
We can take an env var and have a default for it, WDYT?